index.html

```diff
@@ -65,12 +67,12 @@ <h2 class="my-name">I'm Yemsrach Glad to see you!</h2>
     </div>
     </div>
     </section>
-  
-    <section class="work">
-       <section class="work">
+    <!-- Home section start -->
+  <!-- wotk section start -->
+  <section class="work grid-container">
 <div class="snapshoot">
   <img src="./image/Snapshoot Portfolio.png" alt="snapchat">
-  <div class="snapshoot_1"></div>
+  <div class="snapshoot_1">
   <h1>Tonic</h1>
```

Comment:
This tonic text should be wrapped with any of the h2, h3, h4. Kindly replace the current h1. This is also because of semantic.